appserver/concurrent/concurrent-impl/src/main/manpages/org/glassfish/concurrent/admin/create-managed-executor-service.1

```diff
@@ -99,6 +100,15 @@ OPTIONS
            which means that the task queue is essentially unbounded and can
            store any number of submitted tasks.
 
+       --usevirtualthreads
+           If enabled, virtual threads will be used instead of the default
+           thread pool. They provide a quick creation, minimal overhead and
+           fast thread switching, but executed on a limited "carrying threads".
+           Use virtual threads for tasks, which depend on external resources
+           like database, microservices etc. Do not use them for long
+           calculation. Available only on Java 21 and later.
+           The default value is false.
+
```

Comment:
```suggestion
       --usevirtualthreads
           If enabled, virtual threads will be used instead of the default platform
           thread pool. They provide quick creation, minimal overhead, and
           fast thread switching, but are executed on limited "carrying threads".
           Use virtual threads for tasks which depend on external resources
           like databases, micro-services, etc. Do not use them for long
           calculation.
           The default value is false.

```